src/modules/reports/__tests__/reportStore-test.js

```diff
@@ -0,0 +1,70 @@
+import {TestScheduler, Observable} from 'rxjs/Rx';
+import assert from 'assert';
+import {reducer$, loadKDReport} from '../reportStore';
+import {createStore} from 'react-rxjs/dist/RxStore';
+import Report from '../../../models/report';
+
+describe('KDReportStore', () => {
+
+  it('testing reducer', () => {
+    const testScheduler = new TestScheduler((actual, expected) => {
+      console.log(JSON.stringify(actual, null, 2));
+      console.log(JSON.stringify(expected, null, 2));
+      return assert.deepEqual(actual, expected);
+    });
+
+    // mock streams
+
+    const loadKDReportM = '1-----------';
```

Comment:
For eks først laster du. Så clearer du. Testen verifiserer da over tid at når du laster og så clearer så er det du lastet borte.